Source/Spatializer/HoloLens2/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-project(AudioPluginMicrosoftSpatializer)
+project(AudioPluginMicrosoftSpatializerIsac)

```

Comment:
Suggest naming "HoloLens" to match the intent

Reply:
Going to get deleted (oddly I thought I had responded to all these comments yesterday but GitHub is not showing me any).